.github/workflows/lint.yml

```diff
@@ -0,0 +1,29 @@
+name: Lint
+
+on: push
```

Comment:
let's do on all PRs to master:
```
on:
  pull_request:
    branches: [master]
```

Then we can add a branch protection rule to allow merging only if the check passes